src/pages/users/widgets/EditUserForm.vue

```diff
@@ -0,0 +1,115 @@
+<script setup lang="ts">
+  import { PropType, ref, watch } from 'vue'
+  import { SelectOption, useForm } from 'vuestic-ui'
+  import { User, UserRole } from '../types'
+  import UserAvatar from './UserAvatar.vue'
+
+  const props = defineProps({
+    user: {
+      type: Object as PropType<User | null>,
+      default: null,
+    },
+  })
+
+  const newUser = ref<User>({
+    id: -1,
+    avatar: '',
+    fullname: '',
+    role: 'user',
+    username: '',
+    notes: '',
+    email: '',
+    // TODO: Not sure if project must be here
```

Comment:
Yes, let's keep project. We can implement it solely in user, no need to link to project store or something similar.

Reply:
Let's use actual project names though, we can ask gpt to generate data structure

Reply:
Let's deal with projects in different PR when we will deal with projects.